.travis.yml

```diff
@@ -52,9 +52,8 @@ jobs:
       script: yarn test:test-apps
 
     - stage: versioned tests
-      env: EMBER_TRY_SCENARIO=ember-lts-2.8
```

Comment:
dropping support for Ember 2.8 could also be considered a breaking change 🤔

Reply:
Same, this was generated by the default blueprint. Supporting the last 2 LTS releases seems of the 2.X cycle seems very reasonable tho.

Reply:
it is reasonable, yes. but dropping support should still be considered a breaking change and needs a major version bump then.